plugin/pkg/global/finalizerremoval/admission.go

```diff
@@ -80,6 +80,8 @@ func (f *FinalizerRemoval) ValidateInitialization() error {
 	return nil
 }
 
+var _ admission.MutationInterface = &FinalizerRemoval{}
```

Comment:
What about doing all the interface implementation checks like this:
```suggestion
var _ admission.MutationInterface = (*FinalizerRemoval)(nil)
```
I think this is the more widely accepted approach.